### ISSUE TITLE
PusherBackend isn't subclassable

### ISSUE THREAD

**Issue by mattjegan** [aljp/drf_model_pusher#19]
I needed to add some custom logic to `PusherBackend.get_packet` to handle types that weren't json serializable. To do this across my whole project I created a subclass like so:

```python
class MyPusherBackend(PusherBackend):
    def get_packet(self, event, instance):
         ...
```

Then had backends like so:

```python
class MyModelBackend(MyPusherBackend):
     serializer_class = MyModelSerializer
```

This works when you declare `class MyModelBackend(PusherBackend)` however it doesn't work with the extra subclassing of `MyPusherBackend`. The error trace is as follows:

```
Traceback (most recent call last):
  File "/Users/eganm/.local/share/virtualenvs/myproj-xqRGoeqW/lib/python3.6/site-packages/django/utils/autoreload.py", line 228, in wrapper
    fn(*args, **kwargs)
  File "/Users/eganm/.local/share/virtualenvs/myproj-xqRGoeqW/lib/python3.6/site-packages/django/core/management/commands/runserver.py", line 116, in inner_run
    autoreload.raise_last_exception()
  File "/Users/eganm/.local/share/virtualenvs/myproj-xqRGoeqW/lib/python3.6/site-packages/django/utils/autoreload.py", line 251, in raise_last_exception
    six.reraise(*_exception)
  File "/Users/eganm/.local/share/virtualenvs/myproj-xqRGoeqW/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/eganm/.local/share/virtualenvs/myproj-xqRGoeqW/lib/python3.6/site-packages/django/utils/autoreload.py", line 228, in wrapper
    fn(*args, **kwargs)
  File "/Users/eganm/.local/share/virtualenvs/myproj-xqRGoeqW/lib/python3.6/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/eganm/.local/share/virtualenvs/myproj-xqRGoeqW/lib/python3.6/site-packages/django/apps/registry.py", line 116, in populate
    app_config.ready()
  File "/Users/eganm/Documents/myproj/Code/myproj/core/apps.py", line 16, in ready
    from core.pusher_backends import MyModelBackend
  File "/Users/eganm/Documents/myproj/Code/myproj/core/pusher_backends.py", line 10, in <module>
    class MyPusherBackend(PusherBackend):
  File "/Users/eganm/.local/share/virtualenvs/myproj-xqRGoeqW/lib/python3.6/site-packages/drf_model_pusher/backends.py", line 30, in __new__
    model_name = dicts['serializer_class'].Meta.model.__name__.lower()
KeyError: 'serializer_class
```

When I get the chance I'm happy to submit a PR for this.

**Comment by aljp** [comment]
There's something for this!

```
class Meta:
    abstract = True
```

This needs to be documented though

**Comment by mattjegan** [comment]
Fixed by #22
